src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -61,6 +61,7 @@
     private String identifier;
     private Map<String, String> automaticCustomEvents;
     private boolean autoVerificationEmailSuppressed;
+    private List<String> fitBitScopes;
```

Comment:
Not a set?

Reply:
For some strange reason, DynamoDB supports empty lists but not empty sets. I feel like it's cleaner to use "this is never null" semantics other than "this is never empty".